examples/set_attitude_target/set_attitude_target.py

```diff
@@ -1,5 +1,4 @@
-#!/usr/bin/env python
-# -*- coding: utf-8 -*-
+#!/usr/bin/env python2.7
```

Comment:
I'm just wondering if you've ever tested this in Python 3 environment or not.
I modified the code for print function to support Python 3, but I'm not really sure. :)

Reply:
Haven't tested this particular PR alltogether, I just applied a review in #712 that seemed to make sense about the shebang 😸

Reply:
If it works in Python 3 then might make sense to change it to python 3, but either way it seems like a generally safer practice to tie a code to a specific version of Python.

Reply:
Tested to work on both Python2 and Python3.  Changed shebang line.